enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -395,6 +395,8 @@ class AdjointGenerator
           auto diff = Builder2.CreateLoad(
               gutils->invertPointerM(I.getOperand(0), Builder2));
           setDiffe(&I, diff, Builder2);
+        } else {
+          setDiffe(&I, Constant::getNullValue(I.getType()), Builder2);
```

Comment:
Why is this necessary, there shouldn't be a diffe set for inactive values?

Reply:
This was not supposed to be in this PR.